ui/components/app/assets/asset-list/sort-control/index.scss

```diff
@@ -24,4 +24,9 @@
     top: 4px;
     left: 4px;
   }
+
+  .currency-display-component__text,
+  .currency-display-component__suffix {
+    color: var(--color-text-alternative);
```

Comment:
<img width="411" alt="SCR-20241122-nehv" src="https://github.com/user-attachments/assets/8cc9d109-dd1e-4e79-aa59-cf83dc5301a0">

Reply:
Did this with CSS because the CurrencyDisplay component doesn't use DS, and I desperately want to avoid changing that as it's used everywhere.